HackerRank/Python/Data Types/Find second largest number in a list.py

```diff
@@ -1,25 +1,14 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-'''
-Problem Statement
-
+tatement
 Now, lets delve into one of the most basic data types in python, LIST. You are given 'n' numbers. Store them in a list and find the second largest number. Easy enough!
-
 NOTE : Don't use insertion sort
-
 Input Format
-
 First line contains N. Second line contains Array A[] of N integers each separated by a space.
-
 Output Format
-
 Value of the second largest number.
 '''
```

Comment:
typo the comment has no starting point so will lead to error when this code is executed. Also as you created the read me with the links for other challenges , can you do the same thing for this challenge as well so the problem statement can be removed from the code